tasks.yaml

```diff
@@ -59,7 +38,10 @@ tasks:
             namespace: podinfo
             condition: available
 
-  - name: lint-all
-    description: Lint the current repository
+  - name: test-package
+    description: Test the GitLab package from the current branch
```

Comment:
This shouldn't say gitlab since it's just a common test. Assuming this was left over from copy/paste of testing in the gitlab package repo 🤔